admin.go

```diff
@@ -311,13 +311,16 @@ type adminHandler struct {
 // ServeHTTP is the external entry point for API requests.
 // It will only be called once per request.
 func (h adminHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	Log().Named("admin.api").Info("received request",
-		zap.String("method", r.Method),
+	log := Log().Named("admin.api").With(zap.String("method", r.Method),
 		zap.String("host", r.Host),
 		zap.String("uri", r.RequestURI),
 		zap.String("remote_addr", r.RemoteAddr),
-		zap.Reflect("headers", r.Header),
-	)
+		zap.Reflect("headers", r.Header))
+	if r.RequestURI == "/metrics" {
```

Comment:
Should this be an exact match or a prefix match? What about `/metrics/`? (I know it should just be `/metrics` but... figured I'd check.)

Reply:
@mholt using `/metrics/` would result in a `404`, so exact match is the best bet IMO